### PR TITLE
Create manishpandey.json

### PR DESCRIPTION
**Pull request by learnman18**
Create manishpandey.json

## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [ ] The file is in the `domains` folder and is in the JSON format.
- [ ] The file's name is all lowercased and alphanumeric. 
- [ ] You have completed your website. 
- [ ] The website is reachable.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [ ] There is sufficient information at the `owner` field.

### CODE REVIEW

**Comment by learnman18** [comment]
created new JSON for the domain

File: manishpandey.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "learnman18",
+    "email": "manishvirat18.mp@gmail.com"
+  },
+  "record": {
+    "CNAME": "https://www.manishpandey.is-a.dev/"
```
Comment (vintheweirdass) [inline manishpandey.json:7]: eh, you dont want to make your website recursive, right?

`manishpandey.is-a.dev` (this json file) CNAME'd to `manishpandey.is-a.dev` (this file again)
And it doesn't display anything

Comment (vintheweirdass) [reply manishpandey.json:7]: Or if you plan to add the `www` sub-subdomain, add a separate `www.manishpandey.json` and add the content onto it. Like, github pages or smth

File: manishpandey.json
Comment (vintheweirdass) [inline manishpandey.json:1]: place it under the `domains` folder

**Pull request closed without merging**